source/apis/routers .py

```diff
@@ -0,0 +1,50 @@
+from fastapi import HTTPException
```

Comment:
There is a space in the file name.
The file name should be more explicit, descriptive of what router this is, a metadata api. metadata.py, would be a more suitable name.